README.md

```diff
@@ -12,6 +12,11 @@ on how to do that, including how to develop and test locally and the versioning
 
 ## Release Notes
 
+### version TBD
+*Released*: TBD
+(Earliest compatible LabKey version: 20.7)
```

Comment:
20.11?

Reply:
Still works for 20.7 since 20.7 doesn't have the log4j2Version property.

Reply:
ok